samples/file-encryptor/Makefile

```diff
@@ -6,15 +6,42 @@
 OE_CRYPTO_LIB := mbedtls
 export OE_CRYPTO_LIB
 
-all: build
+TARGETS =
+
+SYMCRYPT_TAR = symcrypt-linux-oe_full-AMD64-103.0.1-69dbff3.tar.gz
+SYMCRYPT_URL = https://github.com/microsoft/SymCrypt/releases/download/v103.0.1/${SYMCRYPT_TAR}
+SYMCRYPT_SHA256 = 27C8C1C309B217E74D6055734D305AA86D8CE9E05D6D75AFD12C885F13CEA710
+SYMCRYPT_DIR = SymCrypt
+SYMCRYPT_SO = libsymcrypt.so.103.0.1
+# The linker can only resolve up to single number after .so
+SYMCRYPT_LINK_SO = libsymcrypt.so.103
```

Comment:
Typically, there would be two symlinks:
libsymcrypt.so -> libsymcrypt.so.103
libsymcrypt.so.103 -> libsymcrypt.so.103.0.1

The linker would just refer to libsymcrypt.so and the right version of the shared library would be picked up via the symlinks.